src/api/java/Sort.java

```diff
@@ -27,25 +27,20 @@
 {
     /**
      * Equality operator for objects of type Sort. 
-     * @param o
-     * @return
```

Comment:
Can it be done in a separate pull request/commit? No information is being removed here.